javaparser-core-generators/src/main/java/com/github/javaparser/generator/NodeGenerator.java

```diff
@@ -56,7 +56,7 @@ protected void replaceWhenSameSignature(ClassOrInterfaceDeclaration containingCl
                     throw new AssertionError(f("Wanted to regenerate a method with signature %s in %s, but it wasn't there.", callable.getSignature(), containingClassOrInterface.getNameAsString()));
                 });
     }
-
+    
```

Comment:
Wearing my extra-douche-bag hat I would say not spaces on a blank line. Maybe at some point we could have some automated process remove these things. For now I would not bother changing it.